### DISCUSSION TITLE
Comments in issue removed

### DISCUSSION THREAD

**Discussion by freeseacher** [netbox-community/netbox#9427] [Q&A]
I'm bit confused why comments in https://github.com/netbox-community/netbox/issues/8245 are being constantly removed. 
Am i violating some guides with them ?

**Comment by jeremystretch** [comment]
I've removed your comments because you keep asking for updates, which is very annoying to everyone who receives notifications regarding the issue. Please don't do this. If there is an update, it will appear in the thread.

**Reply by chicks-net** [reply]
From what I see in https://github.com/netbox-community/netbox/blob/develop/CONTRIBUTING.md it does not seem that there is any acceptable way to comment on a feature request to prevent it from being closed for being stale.  I understand your desire to minimize noise, but removing any way for folks to avoid an issue they care about getting closed as stale seems unnecessarily dismissive of your avid community.

**Reply by jeremystretch** [reply]
> it does not seem that there is any acceptable way to comment on a feature request to prevent it from being closed for being stale

Yes, this is by design and [documented in the wiki](https://github.com/netbox-community/netbox/wiki/Issue-Intake-Policy#feature-requests). Any issue which doesn't a) get tagged for milestone assignment (e.g. because of an API change) or b) receive a volunteer to own it eventually goes stale and is closed. This is how we're able to manage 100+ issues per month without amassing an unapproachable backlog of inactive issues.

#8245 is a great example: It's not going stale because it's being ignored. It's going stale because _no one wants to own it_. If @freeseacher had volunteered to own the issue, I'd be happy to assign it to them. Or to anyone else willing to make the commitment.

> but removing any way for folks to avoid an issue they care about getting closed as stale seems unnecessarily dismissive of your avid community.

The reality is that it's not enough for one person to care about an issue, particularly if that person isn't willing to invest the time and effort required. It needs to garner support among the broader community to be appealing to the project. Most feature requests which garner even modest support _do_ eventually make it into the project.

**Reply by freeseacher** [reply]
ok. i see. i've wrote an integration between prometheus and netbox and wondering if i can switch to graphql. it will probably give less load to netbox. instead of using multiple queries to devices and services i will have just one * page count. So i am not an owner neither nor even issue author. Tnx for update. Maybe changing text of issue staleness will give some visibility to problem described above
